### PR TITLE
Update CircleCI and eslint boilerplate.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: lib/tasks/boilerplate.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const buildTree = (root, name) => {
 	return Promise.all([
 		generate('/.circleci/config.yml', template.ciConfigYml()),
 		generate('/.gitignore', template.gitIgnore()),
-		generate('/.eslint', template.eslint()),
+		generate('/.eslintrc.json', template.eslint()),
```
Comment (chee) [inline lib/tasks/boilerplate.js]: !!!!

**Comment by chee** [comment]
will we also move to `origami-ci` in this PR?

**Comment by notlee** [comment]
https://github.com/Financial-Times/origami-build-tools/pull/614#issuecomment-543112071
Ooops, this was supposed to be that 🙈 😴

**Comment by chee** [comment]
hahaha ^___^

**Pull request merged**